psi4/driver/p4util/python_helpers.py

```diff
@@ -1554,43 +1553,3 @@ def _core_triplet(A, B, C, transA, transB, transC):
 core.Matrix.doublet = staticmethod(_core_doublet)
 core.Matrix.triplet = staticmethod(_core_triplet)
 
-
-@staticmethod
-def _core_erisieve_build(
-        orbital_basis: core.BasisSet,
-        cutoff: float = 0.0,
-        do_csam: bool = False
-    ) -> core.ERISieve:
-    """
-    Constructs a Psi4 ERISieve object from an input basis set, with an optional cutoff threshold for
-    ERI screening and an optional input to enable CSAM screening (over Schwarz screening).
-
-    Parameters
-    ----------
-    orbital_basis
-        Basis set to use in the ERISieve object.
-    cutoff
-        Integral cutoff threshold to use for Schwarz/CSAM screening. Defaults to 0.0, disabling screening entirely.
-    do_csam
-        Use CSAM screening? If True, CSAM screening is used; else, Schwarz screening is used. By default,
-        Schwarz screening is utilized.
-
-    Returns
-    -------
-    ERISieve
-        Initialized ERISieve object.
-
-    Example
-    -------
-    >>> sieve = psi4.core.ERISieve.build(bas, cutoff, csam)
-    """
-
-    warnings.warn(
-        "`ERISieve` is deprecated in favor of `TwoBodyAOInt`, and will be removed as soon as Psi4 v1.9 is released.\n",
-        category=FutureWarning,
-        stacklevel=2)
-
-    return core.ERISieve(orbital_basis, cutoff, do_csam)
```

Comment:
you could raise an UpgradeHelper build-48s/src/lib/libint/library-prefix/src/library-build/src/configuration.cc saying how to replace this instead of deleting completely. that is, instead of warning and continuing, it stops with advice. up to you.

Reply:
helps if I send a link to the code rather than a path on my local machine :-)

https://github.com/psi4/psi4/blob/819c588cb7cae7d86637b7727fcd6f5899473b3f/psi4/driver/p4util/python_helpers.py#L824

Reply:
I think this is a pretty good idea! I will give it a look.

Reply:
So I do have a version of the code that raises the `UpgradeHelper` for calling `core.ERISieve.build` instead of deleting `ERISieve` completely, at https://github.com/davpoolechem/psi4/tree/dpoole34/erisieve-upgrade-helper. That is probably better from an API maintenance approach than simply deleting `ERISieve`, but we would probably want to close down this PR and push the other branch instead if we wanted the `UpgradeHelper` approach.

Reply:
I'm thinking a new PR would be easier, so I will likely do that.